mdbook/src/15-interrupts/README.md

```diff
@@ -0,0 +1,66 @@
+
+Let's define an ISR and configure an interrupt to "poke" the MB2 when Button A is pressed
+(`examples/poke.rs`). The board will respond by saying "ouch" and panicking.
+
+```rust
+{{#include examples/poke.rs}}
+```
+
+The ISR handler function is "special". The name `GPIOTE` is required here, indicating
+that this ISR should be stored at the entry for the `GPIOTE` interrupt in the interrupt table.
+
+The `#[interrupt]` decoration is used at compile time to mark a function to be treated specially as
+an ISR. (This is a "proc macro", in case you feel like exploring that concept.)
+
+Marking a function with `#[interrupt]` implies several special things about the function:
+
+* The compiler will check that the function takes no arguments and returns no value. The CPU has no
+  arguments to provide to an ISR, and no place to put a return value from the ISR.
+
+* The compiler will place a vector to this function at the location in the interrupt table
+  implied by the function's name.
+
+* The function will be compiled to finishing by using a return-from-interrupt instruction rather
+  than the normal function return instruction.
+
+* Since the function finishes in a non-standard way, the compiler will understand not to allow
+  directly calling the ISR from normal code.
+
+There are two steps to configure the interrupt. First, the GPIOTE must be set up to generate an
+interrupt when the pin connected to Button A goes from high to low voltage. Second, the NVIC must be
+configured to allow the interrupt. Order matters a bit: doing things in the "wrong" order may
+generate an interrupt before you are ready to handle it.
+
+**Note** As with most microcontrollers, there is a lot of flexibility in when the GPIOTE can generate an interrupt. Interrupts can be generated on low-to-high pin transition, high-to-low (as here), any change ("edge"), when low, or when high. On the nRF52833, interrupts generate an event that must be manually cleared in the ISR to ensure that the ISR is not called a second time for the same interrupt. Other microcontrollers may work a little differently — you should read Rust crate and microcontroller documentation to understand the details on a different board.
+
+When you push the A Button, you will see an "ouch" message and then a panic. Why does the interrupt handler call `panic!()`? Try commenting the `panic!()` call out and see what happens when you push the button. You will see "ouch" messages scroll off the screen. The NVIC records when an interrupt has been issued: that "event" is kept until it is explicitly cleared by the running program. Without the `panic!()`, when the interrupt handler returns the NVIC will (in this case) re-enable the interrupt, notice that there is still an interrupt event pending, and run the handler again. This will continue forever: each time the interrupt handler returns it will be called again. In the next section we will see how to clear the interrupt indication from within the interrupt handler.
```

Comment:
Last sentence: "In the next section...": Technically, how to do this shows up in section 15.2 currently and not 15.1. Maybe replace with "Soon" to keep it in a non-section-number-referring way?